hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/tsconfig.json

```diff
@@ -17,9 +17,10 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "preserve",
+    "jsx": "react",
```

Comment:
This change is done knowingly ? preserve --> react
Any impact to other UI components with this change?

Reply:
This change was tested by @smitajoshi12 . She provided this code for integration test.

Reply:
@sumitagrawl  @devmadhuu 
will do revert back. with new changes devesh will add new commit.